lib/user_lookup.rb

```diff
@@ -1,9 +1,16 @@
 # frozen_string_literal: true
 
 class UserLookup
+  def self.lookup_columns
```

Comment:
Since we're refactoring this, I'm wondering if we should just make this into a constant. Not sure why we need a method call here.

Reply:
It's a public interface now, used by plugins so we'd have to support both for a while, update the plugins and then remove the methods. Not sure if it's worth it.

Reply:
Is this used by plugins via plugin APIs?

Reply:
Used directly:

https://github.com/discourse/discourse-assign/blob/470dd939aa15a1d32e068d7de18c717067b3aecc/app/controllers/discourse_assign/assign_controller.rb#L105

https://github.com/discourse/discourse-assign/blob/470dd939aa15a1d32e068d7de18c717067b3aecc/plugin.rb#L170

tbh, I don't mind these being methods (allows e.g. for easier debug tracing by plopping in a `binding.pry`) but could go either way.

Reply:
Ahh so the ship has already sailed on this. Ignore my comments here then :)